package.json

```diff
@@ -68,6 +68,7 @@
     "serve-dev": "npx concurrently --kill-others npm:serve-live npm:build-js:watch npm:build-css:watch",
     "test": "jest",
     "test:e2e": "testcafe",
+    "test:e2e:base": "testcafe ./tests/e2e/base.test.js",
```

Comment:
in `.testcafe.json` - we crawl for all the test files to run.  I wanted to isolate for locally run tests in Travis CI to only point to this.
Now, since I isolated production tests to be manually run, then we can remove this.

Reply:
done